lib/routes/home/widgets/payments_list/payment_item_avatar.dart

```diff
@@ -29,5 +31,17 @@ class PaymentItemAvatar extends StatelessWidget {
     }
   }
 
-  bool get _shouldShowLeadingIcon => paymentItem.details is LnPaymentDetails &&  (paymentItem.details as LnPaymentDetails).keysend;
+  bool get _shouldShowLeadingIcon {
+    final hasDescription = (paymentItem.description != null &&
+        paymentItem.description!.isNotEmpty);
+    final details = paymentItem.details.data;
+    final isKeySend = (details is LnPaymentDetails) ? details.keysend : false;
+    final isClosedChannelPayment =
+        paymentItem.paymentType == PaymentType.ClosedChannel ||
```

Comment:
PaymentType.ClosedChannel is enough as the condition will return true currently regardless if the channel is pending or closed.